ERROR - 2024-04-29 07:39:45 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'everyday', 3306, '', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4e2\app\Views\inventory.php(48): CodeIgniter\Database\BaseConnection->query('SELECT * FROM p...')
#4 C:\xampp\htdocs\ci4e2\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\ci4e2\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#6 C:\xampp\htdocs\ci4e2\system\Common.php(1171): CodeIgniter\View\View->render('inventory', Array, true)
#7 C:\xampp\htdocs\ci4e2\app\Controllers\Home.php(33): view('inventory')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\Home->inventory()
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4e2\app\Views\inventory.php(48): CodeIgniter\Database\BaseConnection->query('SELECT * FROM p...')
#3 C:\xampp\htdocs\ci4e2\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\ci4e2\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\ci4e2\system\Common.php(1171): CodeIgniter\View\View->render('inventory', Array, true)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\Home.php(33): view('inventory')
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\Home->inventory()
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 07:39:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: home/inventory]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Views\inventory.php(48): CodeIgniter\Database\BaseConnection->query('SELECT * FROM product')
 3 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php')
 4 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('inventory', [], true)
 6 APPPATH\Controllers\Home.php(33): view('inventory')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->inventory()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 08:04:34 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'everyday', 3306, '', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4e2\app\Views\homepage.php(134): CodeIgniter\Database\BaseConnection->query('SELECT * FROM p...')
#4 C:\xampp\htdocs\ci4e2\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\ci4e2\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#6 C:\xampp\htdocs\ci4e2\system\Common.php(1171): CodeIgniter\View\View->render('homepage', Array, true)
#7 C:\xampp\htdocs\ci4e2\app\Controllers\Home.php(13): view('homepage')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\Home->home()
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4e2\app\Views\homepage.php(134): CodeIgniter\Database\BaseConnection->query('SELECT * FROM p...')
#3 C:\xampp\htdocs\ci4e2\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\ci4e2\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\ci4e2\system\Common.php(1171): CodeIgniter\View\View->render('homepage', Array, true)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\Home.php(13): view('homepage')
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\Home->home()
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 08:04:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: home/home]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Views\homepage.php(134): CodeIgniter\Database\BaseConnection->query('SELECT * FROM product')
 3 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\homepage.php')
 4 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('homepage', [], true)
 6 APPPATH\Controllers\Home.php(13): view('homepage')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->home()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 08:19:08 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'everyday' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'everyday', 3306, '', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4e2\app\Views\inventory.php(48): CodeIgniter\Database\BaseConnection->query('SELECT * FROM p...')
#4 C:\xampp\htdocs\ci4e2\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\ci4e2\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#6 C:\xampp\htdocs\ci4e2\system\Common.php(1171): CodeIgniter\View\View->render('inventory', Array, true)
#7 C:\xampp\htdocs\ci4e2\app\Controllers\Home.php(33): view('inventory')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\Home->inventory()
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'everyday' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4e2\app\Views\inventory.php(48): CodeIgniter\Database\BaseConnection->query('SELECT * FROM p...')
#3 C:\xampp\htdocs\ci4e2\system\View\View.php(234): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\ci4e2\system\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\ci4e2\system\Common.php(1171): CodeIgniter\View\View->render('inventory', Array, true)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\Home.php(33): view('inventory')
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\Home->inventory()
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 08:19:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'everyday'
[Method: GET, Route: home/inventory]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Views\inventory.php(48): CodeIgniter\Database\BaseConnection->query('SELECT * FROM product')
 3 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php')
 4 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('inventory', [], true)
 6 APPPATH\Controllers\Home.php(33): view('inventory')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->inventory()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:01:39 --> ParseError: syntax error, unexpected token "<", expecting end of file
[Method: GET, Route: home/add]
in APPPATH\Views\addproduct.php on line 137.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('addproduct', [], true)
 3 APPPATH\Controllers\Home.php(37): view('addproduct')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->add()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:03:26 --> BadMethodCallException: Call to undefined method App\Models\ProductModel::read
[Method: GET, Route: home/inventory]
in SYSTEMPATH\Model.php on line 921.
 1 APPPATH\Views\inventory.php(10): CodeIgniter\Model->__call('read', [])
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('inventory', [], true)
 5 APPPATH\Controllers\Home.php(33): view('inventory')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->inventory()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:03:54 --> BadMethodCallException: Call to undefined method App\Models\ProductModel::read
[Method: GET, Route: home/inventory]
in SYSTEMPATH\Model.php on line 921.
 1 APPPATH\Views\inventory.php(10): CodeIgniter\Model->__call('read', [])
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('inventory', [], true)
 5 APPPATH\Controllers\Home.php(33): view('inventory')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->inventory()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:03:59 --> BadMethodCallException: Call to undefined method App\Models\ProductModel::read
[Method: GET, Route: home/inventory]
in SYSTEMPATH\Model.php on line 921.
 1 APPPATH\Views\inventory.php(10): CodeIgniter\Model->__call('read', [])
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('inventory', [], true)
 5 APPPATH\Controllers\Home.php(33): view('inventory')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->inventory()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:04:38 --> ErrorException: Attempt to read property "product_id" on array
[Method: GET, Route: home/inventory]
in APPPATH\Views\inventory.php on line 35.
 1 APPPATH\Views\inventory.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "product_id" on array', 'C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php', 35)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\ci4e2\\app\\Views\\inventory.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('inventory', [], true)
 5 APPPATH\Controllers\Home.php(33): view('inventory')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->inventory()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:10:22 --> ParseError: syntax error, unexpected token "<", expecting end of file
[Method: GET, Route: home/add]
in APPPATH\Views\addproduct.php on line 137.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('addproduct', [], true)
 3 APPPATH\Controllers\Home.php(37): view('addproduct')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->add()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 09:10:40 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: home/add]
in APPPATH\Views\addproduct.php on line 107.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-04-29 09:10:46 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: GET, Route: home/add]
in APPPATH\Views\addproduct.php on line 107.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-04-29 10:03:59 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:12:48 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:13:15 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:17:07 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:28:12 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:28:29 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:29:11 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:38:48 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:40:18 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:41:19 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "products.php"
[Method: GET, Route: products]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('products.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('products', [], true)
 3 APPPATH\Controllers\ProductController.php(19): view('products', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->read()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:54:42 --> CodeIgniter\Database\Exceptions\DataException: There is no data to insert.
[Method: GET, Route: ProductController/create]
in SYSTEMPATH\BaseModel.php on line 1833.
 1 SYSTEMPATH\BaseModel.php(1833): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\BaseModel.php(806): CodeIgniter\BaseModel->transformDataToArray([], 'insert')
 3 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([], true)
 4 APPPATH\Controllers\ProductController.php(11): CodeIgniter\Model->insert([])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->create()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 10:55:13 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `pr...', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pr...')
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pr...')
#3 C:\xampp\htdocs\ci4e2\system\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pr...', Array, false)
#4 C:\xampp\htdocs\ci4e2\system\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
#5 C:\xampp\htdocs\ci4e2\system\BaseModel.php(1158): CodeIgniter\Model->doDelete(Array, false)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete(Array)
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 10:55:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
[Method: GET, Route: ProductController/delete/12]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 2 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 3 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 4 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:55:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 4 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 5 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 6 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:55:13 --> [Caused by] mysqli_sql_exception: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `product`
WHERE `id` IN (\'12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 5 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 6 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 7 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 10:55:21 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `pr...', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pr...')
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pr...')
#3 C:\xampp\htdocs\ci4e2\system\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pr...', Array, false)
#4 C:\xampp\htdocs\ci4e2\system\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
#5 C:\xampp\htdocs\ci4e2\system\BaseModel.php(1158): CodeIgniter\Model->doDelete(Array, false)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete(Array)
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 10:55:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
[Method: GET, Route: ProductController/delete/12]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 2 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 3 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 4 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:55:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 4 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 5 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 6 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:55:21 --> [Caused by] mysqli_sql_exception: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `product`
WHERE `id` IN (\'12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 5 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 6 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 7 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('12')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 10:55:49 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `pr...', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pr...')
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pr...')
#3 C:\xampp\htdocs\ci4e2\system\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pr...', Array, false)
#4 C:\xampp\htdocs\ci4e2\system\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
#5 C:\xampp\htdocs\ci4e2\system\BaseModel.php(1158): CodeIgniter\Model->doDelete(Array, false)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete(Array)
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 10:55:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
[Method: GET, Route: ProductController/delete/13]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 2 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 3 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 4 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:55:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 4 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 5 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 6 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:55:49 --> [Caused by] mysqli_sql_exception: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `product`
WHERE `id` IN (\'13\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 5 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 6 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 7 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 10:59:26 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `pr...', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pr...')
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pr...')
#3 C:\xampp\htdocs\ci4e2\system\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pr...', Array, false)
#4 C:\xampp\htdocs\ci4e2\system\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
#5 C:\xampp\htdocs\ci4e2\system\BaseModel.php(1158): CodeIgniter\Model->doDelete(Array, false)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete(Array)
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 10:59:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
[Method: GET, Route: ProductController/delete/13]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 2 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 3 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 4 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:59:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 4 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 5 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 6 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 10:59:26 --> [Caused by] mysqli_sql_exception: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `product`
WHERE `id` IN (\'13\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'13\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 5 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 6 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 7 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('13')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-29 11:35:09 --> mysqli_sql_exception: Unknown column 'id' in 'where clause' in C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ci4e2\system\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `pr...', 0)
#1 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `pr...')
#2 C:\xampp\htdocs\ci4e2\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `pr...')
#3 C:\xampp\htdocs\ci4e2\system\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `pr...', Array, false)
#4 C:\xampp\htdocs\ci4e2\system\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
#5 C:\xampp\htdocs\ci4e2\system\BaseModel.php(1158): CodeIgniter\Model->doDelete(Array, false)
#6 C:\xampp\htdocs\ci4e2\app\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete(Array)
#7 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(933): App\Controllers\ProductController->delete('14')
#8 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\xampp\htdocs\ci4e2\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4e2\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4e2\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4e2\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-29 11:35:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
[Method: GET, Route: ProductController/delete/14]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 2 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 3 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 4 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('14')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 11:35:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'14\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'14\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 4 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 5 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 6 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('14')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-29 11:35:09 --> [Caused by] mysqli_sql_exception: Unknown column 'id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('DELETE FROM `product`
WHERE `id` IN (\'14\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `product`
WHERE `id` IN (\'14\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `product`
WHERE `id` IN (\'14\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2823): CodeIgniter\Database\BaseConnection->query('DELETE FROM `product`
WHERE `id` IN :id:', [...], false)
 5 SYSTEMPATH\Model.php(510): CodeIgniter\Database\BaseBuilder->delete()
 6 SYSTEMPATH\BaseModel.php(1158): CodeIgniter\Model->doDelete([...], false)
 7 APPPATH\Controllers\ProductController.php(43): CodeIgniter\BaseModel->delete([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductController->delete('14')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
